README.md

```diff
@@ -1,3 +1,36 @@
-This is a nascent project for implementing a rich text layout engine backed
-by the [swash](https://github.com/dfrg/swash) crate and implementing the 
-[piet](https://github.com/linebender/piet) text API.
+<div align="center">
+
+# Parley
+
+**Rich text layout**
+
+[![Latest published parley version.](https://img.shields.io/crates/v/parley.svg)](https://crates.io/crates/parley)
+[![Documentation build status.](https://img.shields.io/docsrs/parley.svg)](https://docs.rs/parley)
+[![Dependency staleness status.](https://deps.rs/repo/github/linebender/parley/status.svg)](https://deps.rs/repo/github/linebender/parley)
+[![Linebender Zulip chat.](https://img.shields.io/badge/Linebender-%23gpu-blue?logo=Zulip)](https://xi.zulipchat.com/#narrow/stream/205635-text)
+[![Apache 2.0 or MIT license.](https://img.shields.io/badge/license-Apache--2.0_OR_MIT-blue.svg)](#license)
+
+</div>
+
+Parley provides the [Piet](https://github.com/linebender/piet) text API for implementing rich text layout.
```

Comment:
This adapter was removed recently.

Reply:
What would be a better description? Just *Parley provides an API for implementing rich text layout.* ?

Reply:
That seems reasonable to me. We can improve this over time but I’m happy to land this with minimal content to get the correct structure in place.